src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -1312,10 +1312,6 @@ public void Save(ModelSaveContext ctx)
 
         public IDataView Read(IMultiStreamSource source) => new BoundLoader(this, source);
 
-        public IDataView Read(string path) => Read(new MultiFileSource(path));
-
-        public IDataView Read(params string[] path) => Read(new MultiFileSource(path));
```

Comment:
I think we prefer to have two methods because params is a bit slow one, and for performance we prefer to have one method for single file case (which is like 99%) and one for case with multiple file paths.